src/types/countries.ts

```diff
@@ -1,256 +1,260 @@
-	Iran: 'IR';
-	Iraq: 'IQ';
-	Ireland: 'IE';
-	IsleOfMan: 'IM';
-	Israel: 'IL';
-	Italy: 'IT';
-	Jamaica: 'JM';
-	Japan: 'JP';
-	Jersey: 'JE';
-	Jordan: 'JO';
-	Kazakhstan: 'KZ';
-	Kenya: 'KE';
-	Kiribati: 'KI';
-	Korea: 'KR';
-	KoreaDemocraticPeoplesRepublic: 'KP';
-	Kuwait: 'KW';
-	Kyrgyzstan: 'KG';
-	LaoPeoplesDemocraticRepublic: 'LA';
-	Latvia: 'LV';
-	Lebanon: 'LB';
-	Lesotho: 'LS';
-	Liberia: 'LR';
-	LibyanArabJamahiriya: 'LY';
-	Liechtenstein: 'LI';
-	Lithuania: 'LT';
-	Luxembourg: 'LU';
-	Macao: 'MO';
-	Macedonia: 'MK';
-	Madagascar: 'MG';
-	Malawi: 'MW';
-	Malaysia: 'MY';
-	Maldives: 'MV';
-	Mali: 'ML';
-	Malta: 'MT';
-	MarshallIslands: 'MH';
-	Martinique: 'MQ';
-	Mauritania: 'MR';
-	Mauritius: 'MU';
-	Mayotte: 'YT';
-	Mexico: 'MX';
-	Micronesia: 'FM';
-	Moldova: 'MD';
-	Monaco: 'MC';
-	Mongolia: 'MN';
-	Montenegro: 'ME';
-	Montserrat: 'MS';
-	Morocco: 'MA';
-	Mozambique: 'MZ';
-	Myanmar: 'MM';
-	Namibia: 'NA';
-	Nauru: 'NR';
-	Nepal: 'NP';
-	Netherlands: 'NL';
-	NewCaledonia: 'NC';
-	NewZealand: 'NZ';
-	Nicaragua: 'NI';
-	Niger: 'NE';
-	Nigeria: 'NG';
-	Niue: 'NU';
-	NorfolkIsland: 'NF';
-	NorthernMarianaIslands: 'MP';
-	Norway: 'NO';
-	Oman: 'OM';
-	Pakistan: 'PK';
-	Palau: 'PW';
-	PalestinianTerritory: 'PS';
-	Panama: 'PA';
-	PapuaNewGuinea: 'PG';
-	Paraguay: 'PY';
-	Peru: 'PE';
-	Philippines: 'PH';
-	Pitcairn: 'PN';
-	Poland: 'PL';
-	Portugal: 'PT';
-	PuertoRico: 'PR';
-	Qatar: 'QA';
-	Reunion: 'RE';
-	Romania: 'RO';
-	RussianFederation: 'RU';
-	Rwanda: 'RW';
-	SaintBarthelemy: 'BL';
-	SaintHelena: 'SH';
-	SaintKittsAndNevis: 'KN';
-	SaintLucia: 'LC';
-	SaintMartin: 'MF';
-	SaintPierreAndMiquelon: 'PM';
-	SaintVincentAndGrenadines: 'VC';
-	Samoa: 'WS';
-	SanMarino: 'SM';
-	SaoTomeAndPrincipe: 'ST';
-	SaudiArabia: 'SA';
-	Senegal: 'SN';
-	Serbia: 'RS';
-	Seychelles: 'SC';
-	SierraLeone: 'SL';
-	Singapore: 'SG';
-	SintMaarten: 'SX';
-	Slovakia: 'SK';
-	Slovenia: 'SI';
-	SolomonIslands: 'SB';
-	Somalia: 'SO';
-	SouthAfrica: 'ZA';
-	SouthGeorgiaAndSandwichIsl: 'GS';
-	SouthSudan: 'SS';
-	Spain: 'ES';
-	SriLanka: 'LK';
-	Sudan: 'SD';
-	Suriname: 'SR';
-	SvalbardAndJanMayen: 'SJ';
-	Swaziland: 'SZ';
-	Sweden: 'SE';
-	Switzerland: 'CH';
-	SyrianArabRepublic: 'SY';
-	Taiwan: 'TW';
-	Tajikistan: 'TJ';
-	Tanzania: 'TZ';
-	Thailand: 'TH';
-	TimorLeste: 'TL';
-	Togo: 'TG';
-	Tokelau: 'TK';
-	Tonga: 'TO';
-	TrinidadAndTobago: 'TT';
-	Tunisia: 'TN';
-	Turkey: 'TR';
-	Turkmenistan: 'TM';
-	TurksAndCaicosIslands: 'TC';
-	Tuvalu: 'TV';
-	Uganda: 'UG';
-	Ukraine: 'UA';
-	UnitedArabEmirates: 'AE';
-	UnitedKingdom: 'GB';
-	UnitedStates: 'US';
-	UnitedStatesOutlyingIslands: 'UM';
-	Uruguay: 'UY';
-	Uzbekistan: 'UZ';
-	Vanuatu: 'VU';
-	Venezuela: 'VE';
-	Vietnam: 'VN';
-	VirginIslandsBritish: 'VG';
-	VirginIslandsUS: 'VI';
-	WallisAndFutuna: 'WF';
-	WesternSahara: 'EH';
-	Yemen: 'YE';
-	Zambia: 'ZM';
-	Zimbabwe: 'ZW';
-};
+// temporarily disable this to integrate into DCR
 
-export type Country = Countries[keyof Countries];
+export type Country = string;
```

Comment:
I think it makes more sense to just do a string union?

```suggestion
export type Country = Countries[keyof Countries] | string;
```

Reply:
that just ends up being `string`, because all countries are strings too

Reply:
`'AU' | string` will accept any string :)

Reply:
I just meant in terms of the amount of code change, so that in the future, reverting is only removing the union.

Reply:
actually i think you're right, it's a much smaller change